apps/modernization-ui/src/apps/page-builder/generated/models/ValueSet.ts

```diff
@@ -28,5 +28,8 @@ export type ValueSet = {
     valueSetStatusCd?: string;
     valueSetStatusTime?: string;
     valueSetTypeCd?: string;
+    valueSetDescription?: string;
+    type?: string;
+    valueSetName?: string;
```

Comment:
Question: I see changes to a `generated` file but no changes to the API. Was this re-generated based on changes from another PR or were these changes applied manually?